gateway/handler.go

```diff
@@ -233,6 +233,8 @@ func (i *handler) getOrHeadHandler(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	i.handlePathAffinityHints(w, r, contentPath, logger)
+
```

Comment:
As someone not familiar with go nor this codebase, these variable names need work 😳

Reply:
@SgtPooki In Go HTTP server request handlers, `w` and `r` are pretty much convention for `http.ResponseWriter` and  `*http.Request` respectively. This is also seen the Go docs and [examples](https://pkg.go.dev/net/http#example-Handle).